packages/turf-dissolve/test.js

```diff
@@ -1,13 +1,18 @@
 var turfDissolve = require('./index');
 var test = require('tape');
 var polys = require('./test/polys.json');
-var expectedOutput = require('./test/outputPolys.json');
+var expectedOutput = require('./test/outputPolysByProperty.json');
+var expectedOutput2 = require('./test/outputPolysWithNoProperty.json');
```

Comment:
I don't follow on this? What is the point? It seems much trickier than the current approach...